cookbook/attributes/default.rb

```diff
@@ -36,10 +42,19 @@
 
 default['guardian']['repo'] = 'rapid7/guardian'
 
+default['guardian']['database']['user'] = 'guardian'
+default['guardian']['database']['password'] = 'guardian'
+default['guardian']['database']['db_name'] = 'guardian'
+
+default['database']['host'] = '127.0.0.1'
+default['database']['port'] = 3306
+default['database']['user'] = 'root'
+default['database']['password'] = 'change me'
```

Comment:
Should these be `default['guardian']['local_database']` attributes instead so they stay under the `guardian` key? Might make it more obvious they're configuration for the database Guardian connects to, and not overwriting configuration for a "database" cookbook.